Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -125,6 +125,17 @@ void DoState(PointerWrap& p)
   p.Do(ppcState.pagetable_base);
   p.Do(ppcState.pagetable_hashmask);
 
+  if (version >= 133)
```

Comment:
I'm not sure having backwards compatibility for savestates is worth the increased maintenance burden.

Reply:
I can take the commit back out if you want. Anyone else who might have an opinion on this?